server/utils/constants.ts

```diff
@@ -58,3 +58,20 @@ export enum AD_DOC_FIELDS {
 export const MAX_MONITORS = 1000;
 
 export const MAX_ALERTS = 1000;
+
+// TODO: maybe move types/interfaces/constants/helpers shared between client and server
+// side as many as possible into single place
```

Comment:
Any plan on moving more over to the server side?

Reply:
no concrete plan for now, but created issue for this: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/330